bee-message/src/output/alias.rs

```diff
@@ -157,12 +154,14 @@ impl AliasOutput {
     /// Maximum possible length in bytes of the state metadata.
     pub const STATE_METADATA_LENGTH_MAX: usize = 1024;
 
-    ///
-    const ALLOWED_FEATURE_BLOCKS: [u8; 3] = [
-        SenderFeatureBlock::KIND,
-        IssuerFeatureBlock::KIND,
-        MetadataFeatureBlock::KIND,
-    ];
+    /// Returns the set of allowed [`FeatureBlock`]s for an [`AliasOutput`].
+    #[inline(always)]
+    pub fn allowed_feature_blocks() -> FeatureBlockUsages {
```

Comment:
Some of your `allowed_feature_blocks` methods are public and some are not. I don't think this is something that needs to be public, I'd remove it unless proven useful. Also, these can be const methods.

Reply:
I tried making them `const` before, but `bitflags` creates a trait and trait methods can't be marked `const` yet. I'll change the visibility though.